mautic/mautic
# DPMMA-375 Alert variant styles

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ N ]
| New feature/enhancement? (use the a.x branch)      | [ Y ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ N ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR includes: 
- Different styling for warning and error alerts to make them more noticeable.
- Bold and lighter font colour for links in alerts for better visibility
- Make the hover transition brighter instead of darker because the alert background is dark.

Current alert for all types:
![image](https://github.com/mautic/mautic/assets/8580942/6a1553b9-fe4b-43be-b180-56b4d73b1e49)

New Notice (standard):
![image](https://github.com/mautic/mautic/assets/8580942/b0df3093-eaab-4093-a692-e5d7be6eceda)

New Error:
![image](https://github.com/mautic/mautic/assets/8580942/7053cf37-e806-4fcc-ad2f-91ff08b24f03)

New Warning (not sure if is used anywhere now, but I think it could be useful):
![image](https://github.com/mautic/mautic/assets/8580942/ddb45d66-7319-4efa-ac06-e53312d40772)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Test different alert types (Error alert can be triggered using invalid email for contact. I'm not sure how to trigger warning without code changes).
